packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.js

```diff
@@ -593,33 +593,6 @@ describe('ReactDOMFiberAsync', () => {
     expect(containerC.textContent).toEqual('Finished');
   });
 
-  describe('createBlockingRoot', () => {
-    // @gate experimental
-    it('updates flush without yielding in the next event', () => {
```

Comment:
I don't think so because this is blocking specific behavior.

There are 10 tests above for concurrent mode behavior:
<img width="585" alt="Screen Shot 2021-02-26 at 10 25 22 AM" src="https://user-images.githubusercontent.com/2440089/109333690-c04c0100-782d-11eb-9143-80f88e5197fa.png">

Reply:
Yes but doesn't CM "update without yielding" for non-startTransition "default" events too, now? Outside of FB. If yes, do we have a test for this?

It's not quite blocking-specific behavior if we're saying CM inherits some features of blocking mode now.

Reply:
Not yet, we still need to make the changes for default updates to flush sync after a microtask. I'll be sure to add a test for that when we do.